bitrise.yml

```diff
@@ -0,0 +1,51 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  check:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-check.git: { }
+
+  e2e:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-check.git:
+        inputs:
+        - workflow: e2e
+
+  sample:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test.git
+    - TEST_APP_BRANCH: master
+    - BITRISE_PROJECT_PATH: ./_tmp/BullsEye.xcworkspace
+    - BITRISE_SCHEME: BullsEye
+    steps:
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/env bash
+            set -ex
+            rm -rf "./_tmp"
+    - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone.git:
+        inputs:
+        - repository_url: $TEST_APP_URL
+        - branch: $TEST_APP_BRANCH
+        - clone_into_dir: ./_tmp
+    - git::https://github.com/bitrise-steplib/steps-xcode-build-for-test.git@test_plan_output_export:
+        inputs:
+        - project_path: $BITRISE_PROJECT_PATH
+        - scheme: $BITRISE_SCHEME
+        - xcodebuild_options: -testPlan FullTests
+        - destination: generic/platform=iOS Simulator
+    - path::./:
+        title: Step Test
+        inputs:
+        - xctestrun: $BITRISE_XCTESTRUN_FILE_PATH
+        - destination: platform=iOS Simulator,name=iPhone 12 Pro Max
+        - xcodebuild_options: -parallel-testing-enabled YES
+
+  generate_readme:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-readme-generator.git@main:
+        inputs:
+        - contrib_section: docs/contribution.md
+        - example_section: docs/examples.md
```

Comment:
Including these just generates empty Markdown files. Is it worth keeping it here for that?

Reply:
Not really, updated in https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/commits/c87313c8e29a18eff72c568655f34383f583bfef